edgepi-dac.html

```diff
@@ -113,18 +113,18 @@
     <select id="node-input-method">
       <option value="write">Write Voltage</option>
       <option value="read">Read Voltage</option>
-    </select> 
+    </select>
   </div>
   <div class="form-row voltage">
     <label for="node-input-voltage">Voltage to write: </label>
-    <input type="text" id="node-input-voltage" placeholder="0-10" minlength="1" required
-    pattern="^(?:[0-9]|10|\d\.\d{1,4})$">
+    <input type="number" id="node-input-voltage" placeholder="0-10" min="0" max="10" step="0.1"required>
+
   </div>
 </script>
 
 
-<script type="text/html" data-help-name="edgepi-dac-node">
-	<p>EdgePi DAC node that reads/write voltage to a given output channel.</p>
+<script type="text/html" data-help-name="dac">
+	<p>Reads or writes a digital voltage to a given output channel on the EdgePi.</p>
```

Comment:
change this to something like:
```
Generate analog output voltage based on voltage provided.
```
@sjpark608  is this a good description for DAC ?

Reply:
yes, it make sense